lib/scss/theme/_variables.scss

```diff
@@ -0,0 +1,16 @@
+// Colors
+$theme-highlight-color: #e6522c;
+
+// Font
+$font-family: 'Source Sans Pro', sans-serif;
```

Comment:
I thought we were going to use Open Sans font instead of Source Sans Pro, did we decide to switch back to Source Sans Pro?

Reply:
@kaankabalak Deepa asked for me Source Sans Pro. Can you please confirm?

Reply:
Ok, I'll ask her to confirm then